pickle.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Part 1 by Zoe
+#!/usr/bin/env bash
+#unique gender and years experience combinations
+#Usage wages.csv
+
+cat wages.csv | tail -n +2 | tr -s "," | cut -d "," -f 1,2 | sed 's/,/ /' | sort -b  -k1,1 -k2,2n | uniq >>problem.txt
+
+################################################################################
+#Part 2 by Soren
+
+HEADLINE=$(head -n 1 wages.csv |  awk -F ','  '{printf("%s %s %s\n",$1, $2, $4)}')
+HIGHEST=$(tail -n +2 wages.csv | sort -k 4 -t ',' -n | tail -n 1 |  awk -F ','  '{printf("%s %s %s\n",$1, $2, $4)}')
+LOWEST=$(tail -n +2 wages.csv | sort -k 4 -t ',' -n | head -n 1 |  awk -F ','  '{printf("%s %s %s\n",$1, $2, $4)}')
+
+echo "-------------"
+echo "Highest earner"
+echo $HEADLINE
+echo $HIGHEST
+echo "-------------"
+echo "Lowest earner"
+echo $HEADLINE
+echo $LOWEST
+echo "-------------"
+
+TOP10=$(tail -n +2 wages.csv | sort -k 4 -t ',' -n | tail -n 10 |  awk -F ','  '{printf("%s \n",$1)}')
+COUNTER=$((0))
+for GENDER in $TOP10
+do
+  if [ $GENDER == "female" ]
+  then
+    ((COUNTER++))
+  fi
+done
+
```

Comment:
Good. Alternatively, you can also use 
tail -n +2 wages.csv | sort -k 4 -t ',' -n | tail -n 10 |  awk -F ','  '{printf("%s \n",$1)}' | grep -c "female"